classes/class-p4-custom-taxonomy.php

```diff
@@ -216,6 +216,11 @@ public function save_taxonomy_page_type( $post_id ) {
 				return;
 			}
 
+			// Check if post type is PAGE.
+			if ( 'page' === $post->post_type ) {
```

Comment:
Good catch @sagarsdeshmukh !! :+1: I completely missed that.
But can you change it to continue execution only when post_type is 'post' ? 
Since we really only want posts to have terms of this taxonomy.

Reply:
@kirdia Do you mean making it `if ( 'post' != $post->post_type )` ?

Reply:
@sagarsdeshmukh I think he means making it 'post' === $post->post_type

Reply:
Well both will do the job, but I preferred the first one 
```if ( 'post' != $post->post_type ) {return;}```
It makes no big difference.